bin/pycbc_live

```diff
@@ -756,6 +761,8 @@ pycbc.init_logging(args.verbose, format=log_format)
 
 ctx = scheme.from_cli(args)
 fft.from_cli(args)
+padata = livepau.PAstroData(args.p_astro_method, args.p_astro_spec_data,
```

Comment:
In one of the open PRs, I am trying to reduce the number of global variables we use in `pycbc_live`. Could you make `padata` an attribute of the LiveEventManager object instead, initialized in its constructor?